index.js

```diff
@@ -1,18 +1,87 @@
 /***** Deliverable 1 *****/
+
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
+header.style.color = "cyan"
+header.style.backgroundColor = "black"
+header.style.border = "chartreuse"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+
+
+console.log('This is what the traveler object looks like: ', traveler)
+
+const profile = document.querySelector("div#profile")
+profile.innerHTML = `<div id="profile"> 
+
+    <img src="${traveler.photo}"
+      
+    alt="${traveler.name}">
+    
+    <h2>${traveler.name}</h2>
+
+    <em>${traveler.nickname}</em>
+
+    </div>
+
+    <h4>Great Sightings</h4>
+
+    <ul id="animals">
+    <!-- add an <li> for each animal sighting here -->
+    </ul>`
 
 
 /***** Deliverable 4 *****/
 
+// const listOfSightings = document.querySelector("ul#animals")
+
+// traveler.animalSightings.forEach (function (sighting) {
+    
+//     let animalSighting = document.createElement ("li")
+    
+//     animalSighting.dataset.id = sighting.id
+
+//     animalSighting.innerHTML = `<p>${sighting.description}</p>
+
+//     <img src="${sighting.photo}" alt="${sighting.species}"/>
 
+//     <a href="${sighting.link}" target="_blank"> Here's a video about the ${sighting.species} species!</a>
+
+//   </li>`
+
+//   listOfSightings.append(animalSighting)
+ 
+// })
```

Comment:
This looks good. Only thing that I see may cause some strange display is the closing `</li>` tag on line 55